index.html

```diff
@@ -162,7 +162,7 @@
 <script src="https://kit.fontawesome.com/336a975252.js" crossorigin="anonymous"></script>
 <!-- Navbar hamburger funtion-->
 <script>
-    function myFunction() {
+    function hamburgerIcon() {
```

Comment:
A function name should be an "action" and It should describe the functionality, let's say we have to write a function to get developers then a verb like `getDevelopers` is a good fit.

Reply:
got it, Thanks